x-pack/plugins/uptime/public/components/monitor/synthetics/executed_step.tsx

```diff
@@ -73,6 +59,14 @@ export const ExecutedStep: FC<ExecutedStepProps> = ({ step, index, checkGroup })
               />
             </EuiFlexItem>
             <EuiFlexItem>
+              {step.synthetics?.step?.index && checkGroup && (
+                <span>
+                  <StepDetailLink checkGroupId={checkGroup} stepIndex={step.synthetics.step.index!}>
```

Comment:
Nit: it doesn't *seem* like you should need the bang if you've already checked on line 62 that it's defined. I'm still getting used to typescript though.